docs/troubleshooting.en.md

```diff
@@ -54,57 +57,54 @@ We added a hardware IRQ (interrupt request) to the firmware, so when you open yo
+        Then flash the firmware again using Krux-Installer or by typing on the terminal:
 
-`./ktool-linux -B goE -b 1500000 -E`
-
-(Soon **Krux-Installer** will have a full wipe button too)
-
-Then flash the firmware again.
-
-`./ktool-linux -B goE -b 1500000 maixpy_amigo/kboot.kfpkg`
+        ```bash
+        ./ktool-linux -B goE -b 1500000 maixpy_amigo/kboot.kfpkg
+        ```
 
 ### Device didn't reboot, and screen is blank
 
-If the device didn't reboot after successfully flashing the firmware, and the screen is blank after turning it off and on, check if the downloaded file matches the device (this can also occur due to data corruption). Try downloading binaries again. 
+If the device didn't reboot after successfully flashing the firmware, and the screen is blank after turning it off and on, check if the downloaded file matches the device or try downloading binaries again as this can also occur due to data corruption. 
 
-You can also install [MaixPy IDE](https://dl.sipeed.com/shareURL/MAIX/MaixPy/ide/v0.2.5) to help with debugging, Tools > Open Terminal > New Terminal > Connect to serial port > Select a COM port available (if it doesn't work, try another COM port). It will show the terminal and some messages, a message about an empty device or with corrupted firmware appears like: "interesting, something's wrong, boot failed with exit code 233, go to find your vendor."
+You can also install [MaixPy IDE](https://dl.sipeed.com/shareURL/MAIX/MaixPy/ide/v0.2.5) to help with debugging. On its menu go to **Tools -> Open Terminal -> New Terminal -> Connect to serial port -> Select a COM port available** (if it doesn't work, try another COM port). It will show the terminal and some messages, a message about an empty device or with corrupted firmware appears like: "interesting, something's wrong, boot failed with exit code 233, go to find your vendor."
 
 ## **Usage**
 
 ### Why isn't Krux scanning the QR code?
-The level of detail that you see is what Krux sees. If the QR code shown on the device's screen is blurry, the camera lens of the device may be out of focus. It can be adjusted by rotating it clockwise or counter-clockwise to achieve a clearer result. The lenses usually comes with a drop of glue that makes id harder to adjust for the first time. You can use your fingertip, tweezers or small precision pliers to help, being careful to don't damage the fragile lenses.
+
+What you see on the screen is what Krux sees, if the QR code is blurry the camera lens of the device may be out of focus. It can be adjusted by rotating it clockwise or counter-clockwise to achieve a clearer result. The lenses usually comes with a drop of glue that makes id harder to adjust for the first time. You can use your fingertip, tweezers or small precision pliers to help, being careful to don't damage the fragile lenses.
```

Comment:
I see a few typos; corrected with other suggestions, it might read like:

What you see on the screen is what Krux sees through its camera. If the QR code is blurry the camera lens of the device may be out of focus. It can be adjusted by gently rotating the lens clockwise or counter-clockwise to achieve a more clear result. From the factory, the lens may have been adjusted and 'set' with a drop of glue, making it difficult to adjust initially. You can use your fingertip, tweezers or small precision pliers to gently rotate the plastic lens ring.  After the first adjustment, focusing the lens will be easier.

Reply:
Thx Jean! will fix and commit again :smiling_face_with_three_hearts: